<launch>

  <!-- The drive_bot node -->
  <node name="drive_bot" type="drive_bot" pkg="ball_chaser" output="screen">
  </node>

  <!-- The process image node -->
  <node name="process_images" type="process_images" pkg="ball_chaser">
  </node>

</launch>
